schemas/com.snowplowanalytics.snowplow/media_player_quality/jsonschema/1-0-0

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity that is tracked with media player events on data quality change",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_quality",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "bitrate": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "The current bitrate in bps.",
```

Comment:
Might be nice to use `Bits Per Second` rather than `bps`.

Reply:
Replaced `bps` with `bits per second`.